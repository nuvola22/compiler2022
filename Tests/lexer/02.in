'123123''1231'
2
123
2.3
2e3
2e+3
2e-3
1.2e1
1.2e+1
1.2e-2