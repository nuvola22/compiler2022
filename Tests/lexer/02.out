1		1		LitStr			'123123'				'123123'
1		9		LitStr			'1231'				'1231'
2		1		LitInt			2				2
3		1		LitInt			123				123
4		1		LitDob			2,3				2.3
5		1		LitDob			2000				2e3
6		1		LitDob			2000				2e+3
7		1		LitDob			0,002				2e-3
8		1		LitDob			12				1.2e1
9		1		LitDob			12				1.2e+1
10		1		LitDob			0,012				1.2e-2
10		7		Eof							ï¿¿
